tools/crm_mon.c

```diff
@@ -1900,10 +1891,20 @@ clean_up(crm_exit_t exit_code)
     g_clear_error(&error);
 
     if (out != NULL) {
-        if (output_format == mon_output_cgi || output_format == mon_output_html) {
-            handle_html_output(exit_code);
-        } else {
-            out->finish(out, exit_code, true, NULL);
+        switch (output_format) {
+            case mon_output_cgi:
+            case mon_output_html:
+                handle_html_output(exit_code);
+                break;
+
+            case mon_output_console:
```

Comment:
Note: mon_output_console is always defined, even if CURSES_ENABLED is not.  I don't see how it'd be possible to get here if CURSES_ENABLED is not defined, but I can wrap it in an ifdef just in case.